smdebug/tensorflow/utils.py

```diff
@@ -428,4 +428,4 @@ def is_tf_version_greater_than_2_4_x():
 
 
 def is_profiler_supported_for_tf_version():
-    return is_tf_version_2_2_x() or is_tf_version_2_3_x()
+    return is_tf_version_2_2_x() or is_tf_version_2_3_x() or is_tf_version_greater_than_2_4_x()
```

Comment:
Two nits:
- Maybe we can just simplify this and do `return version.parse("2.2.0") <= TF_VERSION`
- Naming schema clashes with `is_tf_version_greater_than_2_4_x`, we should just rename this function to `is_tf_version_greater_than_2_2_x` or rename `is_tf_version_greater_than_2_4_x` appropriately.